mx3_beamline_library/plans/commissioning/examples/scan_1d.py

```diff
@@ -0,0 +1,66 @@
+"""
+This plan runs a 1D Scan. We manually simulate an Ophyd device that
+generates detector data following a Gaussian distribution
+"""
+
+import os
+
+import numpy as np
+from bluesky import RunEngine
+from ophyd import ADComponent
+from ophyd.areadetector.plugins import StatsPlugin_V33
+from ophyd.signal import EpicsSignalRO
+from ophyd.sim import motor1
+from scipy.stats import skewnorm
+
+os.environ["BL_ACTIVE"] = "True"
+# Specify CA ADDRESS list here!
+os.environ[
+    "EPICS_CA_ADDR_LIST"
+] = "01.234.567.89 01.234.567.89"  # Specify CA ADDRESS list here!
+from mx3_beamline_library.devices.classes.detectors import GrasshopperCamera  # noqa
```

Comment:
Remind me why we use the GrasshopperCamera class here and not the Blackfly class? If there is something missing from the blackfly class that we need we should add that.

Reply:
The only reason was that this is the name on CSBS (or at least was when I last checked). I can rename this to blackfly camera